src/panels/NestedPanel.vue

```diff
@@ -25,8 +25,8 @@
               </div>
```

Comment:
Similar to `MinimalPanel`, but without the opacity fade transition since it isn't necessary